template.rb

```diff
@@ -69,14 +69,24 @@
   gem 'truncate_html'
   gem 'view_component', '~> 2.46'
 
+  gem_group :test do
+    # We need to add capybara along with a javascript driver to support the provided system specs,
```

Comment:
```suggestion
    # We need to add capybara along with a javascript driver to support the provided system specs.
```

Period instead of comma?

Reply:
Period is fine 👍 (pushed)